stellarphot/core.py

```diff
@@ -1200,7 +1207,7 @@ def apass_dr9(field_center, radius=1 * u.degree, clip_by_frame=False, padding=10
 
     clip_by_frame : bool, optional
         If ``True``, only return items that are within the field of view
-        of the frame. Default is ``True``.
+        of the frame.
```

Comment:
You removed the statement that the default was ``True``.  Do you want to instead say it is ``False``.